frontend/website/pages/docs/node-operators/snark-workers.mdx

```diff
@@ -0,0 +1,116 @@
+import Page from '@reason/pages/Docs';
+export default Page({title: "Snark Workers for Node Operators"});
+
+## Intro
+
+While most protocols have just one primary node operator class (often called miners, validators, or block producers), Coda has a second class — the **snark worker**.
+
+Snark workers are integral to the Coda network's health because these nodes are responsible for snarking, or producing SNARK proofs, of transactions in the network. By producing these proofs, snark workers help maintain the succinctness of the Coda blockchain.
+
+In this document, we'll briefly cover why snark workers are needed, how the economic incentives align, and provide operational details of performing snark work. Feel free to click through to any of the sections above that are most relevant to your needs.
+
+Note: This document primarily targets node operators, and will not cover the theory of zk-SNARKs. Deep knowledge of SNARKs is not required to read this section, but it will be helpful to know roughly how SNARKs work and what they are useful for. If this is not the case, please check out [this primer on SNARKs](https://codaprotocol.com/blog/zkp-guide.html) first.
+
+## Why Snark Workers?
```

Comment:
I wonder if this + Snarking Transactions section are more information than technically necessary for node operators, though interesting to someone wanting to know more details about the protocol, might fit better in the Coda Protocol Architecture section of the docs

Reply:
The impetus for this to be written was pretty consistent questions from node operators curious how these bits worked which is why it was initially nested under here. But maybe we can move what exists to protocol architecture and that way we can delay writing that last section